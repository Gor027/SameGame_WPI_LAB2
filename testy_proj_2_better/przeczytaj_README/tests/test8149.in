L[ 
L [[
 [L  L [L[L[L L[ LL[LL  L    [L[[[L[[L [[ L [ LLLL L[L[[ [L
[[LL [ L[[[L[   [L 
 [[[ L[L[  L  L[[L[ [LL[ [[L [[  L [LL [L[[ L    [[[
LL[ L [ LL[L [LL[ L L[LL[[LLL  [[[L  [[ [[L[LL [  LL[L[L [L  LL[L L
[[ LL[[[ L[LL L[L[[[ [[ [[L [L[ [L [ [[[ L[ L[LL [   L[LL[ [[L L
   L[LL[[ [L [LLL L LL [ L  L   [  [LLL[[[[[  [LLL 
