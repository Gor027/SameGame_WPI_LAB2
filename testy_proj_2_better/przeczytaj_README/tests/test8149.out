           [
 [ [ L [L LL  L[[ L  L               L    L
L[ L [[L[[[L  L[[L[ LL   [      [[ [ L [[[[ [  L L
L[[[LL[L[[[LL LL[LL [[L[L[[ L[L [L [[[[[L[L L L[ [ L L[[ [
[[LLL[[[LL[L[[L[LL[[L[LL[[LL[[[[[[L[L[[[[L[[LLLLL[[[[L[L [L  L [
LLLL[LL[[L[LL[LLL[L[LL[[[LLLLL[[[LL[LLL[[[[[L[[LLLLL[L[LL[L[[LLLLL
